operatoringress/v1/0000_50_dns-record.yaml

```diff
@@ -13,7 +13,7 @@ spec:
     listKind: DNSRecordList
     plural: dnsrecords
     singular: dnsrecord
-  scope: ""
+  scope: Namespaced
```

Comment:
You're changing the scope of a released CRD, @sttts is it backwards compatible? I know it looks one-liner, but you're changing basic category of a resource. This might cause problems, especially that this was already pre-existing. 
/hold
for more information

Reply:
I think `Namespaced` is the default value in CRD v1beta1:

https://github.com/kubernetes/apiextensions-apiserver/blob/master/pkg/apis/apiextensions/v1beta1/types.go#L55-L57

Without this PR, we cannot apply the CRD to a cluster since in crd v1, scope cannot be empty.

Reply:
That's fair.